extensions/wikia/ArticleComments/classes/ArticleCommentList.class.php

```diff
@@ -505,34 +505,22 @@ public function blockedPage() {
 			return '';
 		}
 
+		$block = $wgUser->mBlock;
+
 		list( $blockerName, $reason, $ip, $blockid, $blockTimestamp, $blockExpiry, $intended, $isGlobal ) = [
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$isGlobal" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)